docs/neural/solvers.rst

```diff
@@ -0,0 +1,24 @@
+ott.neural.solvers
+==================
+.. module:: ott.neural.solvers
+.. currentmodule:: ott.neural.solvers
+
+Under reconstruction. .. TODO(marcocuturi): add some nice text here please.
```

Comment:
@marcocuturi also here, thank you!

Reply:
This module implements various solvers to estimate optimal transport between two probability measures, through samples, parameterized as neural networks. These neural networks are described in :mod:`~ott.neural.models`, borrowing lower-level components from :mod:`~ott.neural.layers`, building from the
`flax <https://flax.readthedocs.io/en/latest/examples.html>`__ library.